hw/formal/formal_conn.sh

```diff
@@ -26,7 +26,7 @@ set -e
 CORE_PATH=systems:top_earlgrey
 REPO_PATH=$(readlink -f ../..)
 
-export TOP="top_earlgrey"
+export DUT_TOP="top_earlgrey"
```

Comment:
Whoops, I might have broken this in my earlier PR.

Reply:
No no it is probably me.. I have too much changes in my local.. I think it was my mistake..